pkgs/development/python-modules/thefuzz/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "thefuzz";
+  version = "0.19.0";
```

New packages should be limited to supported Python releases as the chance is high that they are not used/tested on deprecated releases.

```suggestion
  version = "0.19.0";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```